packages/@okta/i18n/dist/properties/login.properties

```diff
@@ -229,8 +229,8 @@ factor.password = Password
 factor.customFactor.description = Redirect to a third party MFA provider to sign in to Okta.
 factor.customFactor.description.generic = Redirect to a third party MFA provider to sign in.
 factor.customFactor.description.specific = Redirect to a third party MFA provider to sign in to {0}.
-factor.webauthn = Security Key or Built-in Authenticator
-factor.webauthn.description = Use a security key (USB or bluetooth) or a built-in device authenticator (Windows Hello, macOS TouchID, etc.)
+factor.webauthn.v2 = Security Key or Biometric Authenticator
```

Comment:
nit: `v2` sounds like a string for webauthn v2, as opposed to just a new string (there is now webauthn v2, I know)... maybe `factor.webauthn.biometric`?? same nit for other key changes